package.json

```diff
@@ -96,7 +96,7 @@
     "postcss-loader": "^7.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "storybook": "7.0.27",
+    "storybook": "7.4.0",
```

Comment:
This fixes the bug, thanks for spotting it!

However, as Storybook is not a dependency (for production code) it can also be moved into the devDependencies block.

Reply:
sorry for confusing you, i actually overlooked that this is actually the devDependencies block